packaging/deb/erigon/DEBIAN/postinst

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# This is a postinstallation script so the service can be configured and started when requested
+#
+sudo adduser --disabled-password --disabled-login --shell /usr/sbin/nologin --quiet --system --no-create-home --home /nonexistent erigon
+if [ -d "/opt/erigon" ]
+then
+    echo "Directory /opt/erigon exists."
+else
+    mkdir -p /opt/erigon
+    sudo chown -R erigon /opt/erigon
+fi
+sudo systemctl daemon-reload
```

Comment:
also a lot of sudo here, and no new line at end of debian files

Reply:
👍 will address

Reply:
Updated